### ISSUE TITLE
Fieldtype registering new redirect on empty field

### ISSUE THREAD

**Issue by michaelroper** [nystudio107/retour#8]
Since updating to v1.0.8, if I edit an existing entry, and don't enter anything in the url pattern field, it still saves a new dynamic redirect with an empty pattern. This means that when I edit another entry and the same thing happens, it errors because a url pattern `''` already exists in the database, and won't save the entry. Error is as follows:
```Internal Server Error
CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'craft_retour_redirects_redirectSrcUrlParsed_unq_idx']```

**Comment by khalwat** [comment]
I guess I'll have it ignore/delete redirects if the redirectSrcUrl is empty

**Comment by khalwat** [comment]
This is fixed in 1.0.9.  It'll delete a redirect if you save it with an empty Legacy URL Pattern.  It'll also fail gracefully if somehow you manage to save two redirects with the same Legacy URL Pattern.
